src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -804,25 +921,25 @@ subroutine mixedlayer_restrat_Bodner(CS, G, GV, US, h, uhtr, vhtr, tv, forces, d
   ! Estimate w'u' at h-points
   do j = js-1, je+1 ; do i = is-1, ie+1
     w_star3 = max(0., -bflux(i,j)) * BLD(i,j) & ! (this line in Z3 T-3 ~> m3 s-3)
-              * ( ( US%Z_to_m * US%s_to_T )**3 ) ! m3 s-3
+              * ( ( US%Z_to_m * US%s_to_T )**3 ) ! [m3 T3 Z-3 s-3 ~> 1]
     u_star3 = ( US%Z_to_m * US%s_to_T * U_star_2d(i,j) )**3 ! m3 s-3
     wpup(i,j) = max( CS%min_wstar2, &           ! The max() avoids division by zero later
                 ( CS%mstar * u_star3 + CS%nstar * w_star3 )**two_thirds ) & ! (this line m2 s-2)
-                * ( ( US%m_to_Z * US%T_to_s )**2 ) ! Z2 T-2 ~> m2 s-2
+                * ( US%m_to_L * GV%m_to_H * US%T_to_s**2 ) ! [L H s2 m-2 T-2 ~> 1 or kg m-3]
```

Comment:
I'm not sure why the units for the Boussinesq version changed. Were they wrong? I thought you'd been through and fixed them already?

Reply:
w'u' is a vertical flux of horizontal momentum. The units before [Z2 T-2] were wrong - they should have been [L Z T-2], but this was compensated for by conversion factors back and forth between L and Z units, first when converting the wind stresses into a friction velocity (outside of this routine) and then the reverse when w'u' is used to set the streamfunction on lines 893 and 934 of the original version of the code.  Setting the units properly lets us eliminate some factors of `US%L_to_Z`.